composer.json

```diff
@@ -25,6 +25,9 @@
     }],
     "require": {
         "php": ">=5.4",
+        "ext-ctype": "*",
```

Comment:
ctype really needed for all db types?

Reply:
It seems like this is actually a package dependency because when we try to create a new migration without Ctype the following error is shown:

```
/app $ vendor/bin/phinx create NewMigration
Phinx by CakePHP - https://phinx.org. 0.8.1


Fatal error: Uncaught Error: Call to undefined function Symfony\Component\Config\ctype_alpha() in /app/vendor/symfony/config/FileLocator.php:87
Stack trace:
#0 /app/vendor/symfony/config/FileLocator.php(42): Symfony\Component\Config\FileLocator->isAbsolutePath('phinx.php')
#1 /app/vendor/robmorgan/phinx/src/Phinx/Console/Command/AbstractCommand.php(218): Symfony\Component\Config\FileLocator->locate('phinx.php', '/app', true)
#2 /app/vendor/robmorgan/phinx/src/Phinx/Console/Command/AbstractCommand.php(236): Phinx\Console\Command\AbstractCommand->locateConfigFile(Object(Symfony\Component\Console\Input\ArgvInput))
#3 /app/vendor/robmorgan/phinx/src/Phinx/Console/Command/AbstractCommand.php(93): Phinx\Console\Command\AbstractCommand->loadConfig(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#4 /app/vendor/robmorgan/phinx/src/Phinx/Console/Command/Create.php(154): Phinx\Console\Command\AbstractCommand->bootstrap(Object(Symfony\Component\Console\Input\ArgvInput), Objec in /app/vendor/symfony/config/FileLocator.php on line 87
```

Reply:
The PHP manual says ctype functions should be enabled by default http://php.net/manual/en/ctype.installation.php. Based on the comments only FreeBSD seems to have a problem.

Reply:
So it would be safer to also put them into suggest for now IMO.